### DISCUSSION TITLE
Docker instance failing to connect to server (WS)?

### DISCUSSION THREAD

**Discussion by KrishnaPG** [TuGraph-family/tugraph-db#836] [Q&A]
I am following the steps outlined [here](https://tugraph-db.readthedocs.io/en/v4.1.0/3.quick-start/1.preparation.html#fast-experience-through-docker) to setup the docker instance. 
```sh
docker run -d -p 7070:7070 -p 9090:9090 -p 7687:7687 --name tugraph_demo tugraph/tugraph-runtime-centos7 lgraph_server
```
The login screen apears, but the login fails. Not sure why
![image](https://github.com/user-attachments/assets/00c92a3f-b86d-4cbe-b511-f032ecaf66e1)

Any help on how to login correct would be of great help. I am using the latet docker image 4.5.1 version

**Comment by KrishnaPG** [comment]
Websocket connection is giving error: "buffer overflow". please see the screenshot below

![image](https://github.com/user-attachments/assets/7ca4881b-4539-492d-b993-b71ecdb07377)

**Reply by ljcui** [reply]
Please use chrome and have a try.
